html_node.go

```diff
@@ -159,7 +162,9 @@ func (pn *ParsedNode) SelectorPriority(sel Selector) int16 {
 		}
 	}
 	if sel.Tag == pn.Tag {
-		priority += 1
+		priority += 2
```

Comment:
do we have a sense of why this is plus 2? is it due to initial priority matching global priority if there is a matched tag?

Reply:
per discussion in discord: this is a magic number that gets tests passing again and we dont have strong knowledge or opinions as to where this magic value originates from